src/pages/api/graphql.tsx

```diff
@@ -110,7 +112,8 @@ export const apolloServer = new ApolloServer({
       }
 
       // Set db connection, keystone api, and user in context
-      return { db, keystoneUrl, user: loggedInUser }
```

Comment:
realized that since we use data sources now, we do not need to access the keystone URL in the resolvers 🎉

Reply:
Nice!